### PR TITLE
chore: Load keyvault url from environment variable in example

### PR DESCRIPTION
**Pull request by karataliu**
**Reason for Change**:
Make it easy to test in sovereign clouds

**Please answer the following questions with yes/no**:

Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?

- [ ] yes
- [x] no

**Notes for Reviewers**:

### CODE REVIEW

**Review by aramase — APPROVED** [review]

File: examples/msal-go/main.go [hunk 1]
```diff
@@ -12,10 +12,13 @@ import (
 )
 
 func main() {
-	keyvaultName := os.Getenv("KEYVAULT_NAME")
-	secretName := os.Getenv("SECRET_NAME")
-
-	keyvaultURL := fmt.Sprintf("https://%s.vault.azure.net/", keyvaultName)
+	keyvaultURL := os.Getenv("KEYVAULT_URL")
+        if keyvaultURL == "" {
+                keyvaultName := os.Getenv("KEYVAULT_NAME")
```
Comment (aramase) [inline examples/msal-go/main.go:17]: Could you move `keyvaultName` outside the if loop?

**Review by aramase — CHANGES_REQUESTED** [review]

File: examples/msal-go/main.go (hunk 1 above)
Comment (karataliu) [reply examples/msal-go/main.go:17]: build fixed. ptal

Comment (aramase) [reply examples/msal-go/main.go:17]: > I suppose we can remove keyvaultName in future after the doc is changed to use keyvaultURL?

For this change, we should update the env var used across all the language examples here: https://github.com/Azure/azure-workload-identity/tree/main/examples

**Review by aramase — APPROVED** [review]

**Pull request merged**